web/src/datavzrd.js

```diff
@@ -1115,6 +1081,85 @@ export function load() {
                     line_numbers("none");
                 }
             })
+
+            $('#downloadcsv-btn').on('click', function() {
```

Comment:
I would prefer to keep the `function downloadCSV()` and then refer to it from within the event listener. That keeps the code a bit more clean. Same goes for the other function.